proto/api.proto

```diff
@@ -53,9 +53,147 @@ service Api {
 
   // Drift V2
 
+  rpc PostCloseDriftPerpPositions(PostCloseDriftPerpPositionsRequest) returns (PostCloseDriftPerpPositionsResponse) {
+    option (google.api.http) = {
+      post: "/api/v2/drift/perp/close"
+      body: "*"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      description: "Returns a partially signed txn signature to close a perp position on Drift";
+      summary: "closes a perp position on Drift";
+      tags: ["Trade", "Perp", "Orderbook"];
+    };
+  }
+
+  rpc GetDriftPerpOrderbook(GetDriftPerpOrderbookRequest) returns (GetDriftPerpOrderbookResponse) {
+    option (google.api.http) = {
+      get: "/api/v2/drift/perp/orderbook/{contract}"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      description: "Returns perp contract's orderbook on Drift. Use limit param to reduce the number of bids/asks returned";
+      summary: "Perpetuals contract orderbook on Drift";
+      tags: ["v2", "Perp", "Orderbook", "Market"];
```

Comment:
I think we should have less tags as this will make the swagger very confusing. How about [ "Drift V2", "Drift Market"] . The endpoint appears under every specified tag in the swagger doc so we should make it less repetitive. 
Same rule should applied for all v2 endpoints. Only a couple tags that will be different than the ones used for V1. It's ok to use 2 words for a tag, i.e. "Drift V2"